contributors/devel/sig-node/separate-mount-namespace.md

```diff
@@ -0,0 +1,199 @@
+secrets or other protected container mount contents. Locking access to even
+listing the mountpoints behind the ability to enter the mount namespace (which
+in turn requires `CAP_SYS_CHROOT` and `CAP_SYS_ADMIN`) removes one level of
+potential access to container-specific mounts.
+
+### No one uses it (that we know of)
+
+After much investigation and conversation on both the sig-node and sig-storage
+mailing lists, as well as within Red Hat's OpenShift communities, we have yet
+to turn up a use case that relies on this container-to-host propagation.
+
+### No one _should_ use it
+
+In fact, it may be seen as an anti-pattern and run contrary to the
+container-native way of doing things. Relying on a host OS to interact with a
+container-owned mountpoint seems suspect.
+
+### Testing implications
+
+With this requirement relaxed, it would be possible to run multiple instances
+of Kubernetes on a single host without the various mountpoints interfering with
+one another. There may be other inter-instance difficulties besides the mount
+namespace, but this does move in that direction without resorting to more
+complicated solutions like trying to run Kubernetes inside of containers.
+
+## Costs and Implications
+
+If a Kubernetes distribution intends to implement this mount-hiding mode of
+operation, there are a number of costs and implications that should be
+documented for system administrators.
+
+### Privileged pods
+
+Privileged pods that previously expected to have direct access to the host's
+root mount namespace (such as the one spawned by `oc debug`, will now be rooted
+in the new container-runtime mount namespace.  Shells or utilities in
+priviliged pods like this can enter the host OS namespace by running `nsenter
+-t 1 -m` (assuming they have the host OS's /proc filesystem mounted so they can
+reach the mount namespace of the real init process in /proc/1/ns/mnt)
+
+### Legacy Tools
+
+Any monitoring tool that relies on the ability to run in the host OS and have
+visibility of mountpoints created by Kubelet, the Container Runtime, or
+containers themselves, will need to enter the container mount namespace in
+order to see these mountpoints.  This could be done by:
+
+- Altering the legacy tool to be cloud-native
+- Altering the scripts that launch the legacy tool to run it inside the
+  container-mount namespace
+- Altering the legacy tool to be aware of the container-mount namespace and
+  enter it as needed to probe or interact with the container-specific
+  mountpoints
+- Providing a system-wide mechanism to revert the Kubernetes installation back
+  to the original "all mounts in the Host OS" mode of operation.
+
+### Administrative Access
+
+An administrator logged in to the host OS wishing to obtain information about
+the Kubernetes system for debugging or auditing purposes will need to be aware
+of the new mount namespace, and will need to run 'nsenter' to view or interact
+with these mountpoints.
+
+## Implementation Suggestions
```

Comment:
@lack Let's add an explicit note that there are no changes proposed or needed to kubelet or other components to use this new style of implementation.